url/urltestdata.json

```diff
@@ -5305,5 +5305,51 @@
     "pathname": "joe@example.org,2001:foo/bar",
     "search": "",
     "hash": ""
+  },
+  "# file URLs with repeated slsashes by Daijirō Wachi",
+  {
+    "input": "file:////////",
+    "base": "about:blank",
+    "href": "file:///",
+    "origin": "null",
```

Comment:
Origin should be omitted for file URL tests because of https://url.spec.whatwg.org/#origin's "exercise for the reader"